tests/real-examples/real-examples-results.yaml

```diff
@@ -1983,11 +1983,6 @@ https://stackoverflow.com/questions/26575587:
         </code>
       type: pypi
       name: pip
-    - range: |-
-        <code>pip install scipy-0.16.1-cp27-none-win_amd64.whl
-        </code>
-      type: pypi
-      name: scipy-0.16.1-cp27-none-win_amd64.whl
```

Comment:
@OshriAsulin please review this change made by me.

Make sure all the removed results are indeed wrong results, and all the added are correct.